src/pages/Popup/components/AlarmUp.jsx

```diff
@@ -51,6 +51,10 @@ class Alarmview extends Component {
     }
     this.setState({ alarms: setalarms });
   }
+  
+  trunc(string,num) { 
```

Comment:
add a space between `string` and `num` variable as good dev practise

Reply:
@neeraj-2 Use prettier as it formats the code nicely